releases/cloud/point-2024-02-22-release-note.md

```diff
@@ -13,29 +13,33 @@ description: >-
 * **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
 * **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
 
-
-## Improvements & Bug Fixes
+## Features
 
 * **The Change Storage Limit action** was added to the Storage Metrics report.
+  * You can now customize and change the storage limit for each site, meaning that if you notice a certain site has a larger influx of data than others, you can increase the storage limit for that site while lowering the storage limit for others. 
+  * The cumulative storage for all your sites cannot exceed the total storage available for your tenant.
+
+
+
+## Improvements & Bug Fixes
 
 * **The Remove Access action was enabled** for users in the Access review task on the Sharing step - Users View.   
   * Multiselection of users is also enabled.
 
-* **Every file** now has the sensitivity and retention labels applied when they are available. 
+* **Every file with a sensitivity or retention label** assigned in Microsoft 365 will now have the labels applied within Syskit Point as well.
```

Comment:
ovo ne spominjati. to je za sad samo u bazi. ne prikazuje se to još nigdje pa niti nećemo spominjati.